test/distributed/_composable/test_fsdp_logging.py

```diff
@@ -0,0 +1,58 @@
+# Owner(s): ["module: fsdp"]
```

Comment:
Let us move this to be `test/distributed/_composable/fsdp/test_fully_shard_logging.py`

Reply:
Will move there in the next version